src/main/java/org/sonatype/gradle/plugins/scan/nexus/iq/scan/NexusIqScanTask.java

```diff
@@ -131,6 +129,31 @@ public void scan() {
     }
   }
 
+  private void verifyOrCreateApplication(InternalIqClient iqClient) throws IqClientException {
+    if (StringUtils.isNoneBlank(extension.getOrganizationId(), extension.getApplicationId())) {
+      NexusIqApi nexusIqApi = NexusIqApi.build(extension.getServerUrl(), extension.getUsername(),
+          extension.getPassword(), buildUserAgent());
+
+      boolean applicationExists = nexusIqApi
```

Comment:
Question. Before checking if the application exists, is it worth checking the organisation id is valid and actually refers to an existing organisation, and throw an exception if it doesn't? Otherwise is it possible to create an application for an org which doesn't exist?

Reply:
> Question. Before checking if the application exists, is it worth checking the organisation id is valid and actually refers to an existing organisation, and throw an exception if it doesn't? Otherwise is it possible to create an application for an org which doesn't exist?

When seting an unexisting organization ID the operation fails and the user gets this output in console/terminal (using `667371c75ffd4662a9b6d4ee1f5f0d3d111` as example of an invalid org id):
```
Could not scan the project: [404 Not Found] during [GET] to [http://localhost:8070/api/v2/applications/organization/667371c75ffd4662a9b6d4ee1f5f0d3d111] [NexusIqApi#getApplicationsByOrganizationId(String)]: [Cannot find organization with ID 667371c75ffd4662a9b6d4ee1f5f0d3d111.]
```